research/operator_author_interviews/index.md

```diff
@@ -0,0 +1,100 @@
+---
+parent: Research
+
```

Comment:
just a general question: How do we plan to align the research efforts shown in the repo with releases like we do with all the designs?